common/bits.cc

```diff
@@ -0,0 +1,51 @@
+/*
+ *  nextpnr -- Next Generation Place and Route
+ *
+ *  Copyright (c) 2013  Mike Pedersen
```

Comment:
Just to check, this code is OK to be under the ISC license?

Reply:
Ya, it was under MIT, hence the copyright notice.